eco-core/src/main/react/clients/utils.ts

```diff
@@ -20,3 +20,13 @@ export const validateResponse = <T>(
     throw new Error(text)
   })
 }
+
+export type QueryParameters = {
+  [key: string]: any
+}
+
+export const toQueryString = (object: object): any =>
+    Object.entries<any>(object)
+        .filter(([_, value]) => value != null)
```

Comment:
I moved this method, but also flipped the null check, it filtered out non-null instead of null before, resulting in an empty query string in most cases.